# Methicillin-resistant Staphylococcus aureus in Saudi Arabia: genomic evidence of recent clonal expansion and plasmid-driven resistance dissemination

## Abstract


__Objectives__: Staphylococcus aureus is a leading cause of hospital-acquired infections worldwide. Over recent decades, methicillin-resistant Staphylococcus aureus (MRSA), which is resistant to multiple antimicrobials, has emerged as a significant pathogenic strain in both hospital and community settings. The rapid emergence and dissemination of MRSA clones are driven by a dynamic and evolving population, spreading swiftly across regions on epidemiological time scales. Despite the vast geographical expanse and diverse demographics of the Kingdom of Saudi Arabia and the broader West Asia region, the population diversity of MRSA in hospitals in these areas remains underexplored.

__Methods__: We conducted a large-scale genomic analysis of a systematic Staphylococcus aureus collection obtained from 34 hospitals across all provinces of KSA, from diverse infection sites between 2022 and 2024. The dataset comprised 582 MRSA and 30 methicillin-susceptible Staphylococcus aureus (MSSA) isolates, all subjected to whole-genome sequencing. A combination of phylogenetic and population genomics approaches was utilized to analyze the genomic data. Hybrid sequencing approach was employed to retrive the complete plasmid content.

__Results__:The population displayed remarkable diversity, comprising 35 distinct sequence types (STs), with the majority harboring community-associated SCCmec loci (types IVa, V/VII, and VI). Virulence factors associated with community-acquired MRSA (CA-MRSA), including Panton-Valentine Leukocidin (PVL) genes, were identified in 12 distinct STs. Dominant clones, including ST8-t008 (USA300), ST88-t690, ST672-t3841, ST6-t304, and ST5-t311, were associated with infections at various body sites and were widely disseminated across the country. Linezolid and vancomycin resistance were mediated by cfr-carrying plasmids and mutations in the vraR gene (involved in cell-wall stress response) and the murF gene (peptidoglycan biosynthesis) in five isolates, respectively. Phylodynamic analysis revealed rapid expansion of the dominant clones, with their emergence estimated to have occurred 10â€“20 years ago. Plasmidome analysis uncovered a diverse repertoire of blaZ-containing plasmids and the sharing of erm(C)-encoding plasmids among major clades. The acquisition of plasmids coincided with clonal expansion.

__Conclusions__:Our results highlight the recent concurrent expansion and geographical dissemination of CA-MRSA clones across hospitals. These findings also underscore the interplay between clonal spread and horizontal gene transfer in shaping the resistance landscape of MRSA.

## Files and Descriptions

| File Name                                  | Description                                                      |
|--------------------------------------------|------------------------------------------------------------------|
| Metadata_MOH_MRSA_19Jan.csv                | Contains genetic, demographic, and phenotypic metadata           |
| *_02_09_2024_2306.results.csv              | Scoary results for comparative genomics analysis                 |
| gene_presence_absence_roary.csv            | Pangenome file generated by panaroo                              |
| AMR__genes__CARD_v3.0.8_SRST2__results.txt | SRST2 results for antimicrobial resistance genes detection       |
| Staphylococcus_VF_clustered__results.txt   | SRST2 results for virulence factor genes detection               |
| run_panaroo.sh                             | Shell script to run Panaroo for pangenome analysis               |
| run_scoary.sh                              | Shell script to run Scoary for gene trait association analysis   |
| tmp_resistance.sh                          | Shell script for run SRST2 to detect resistance genes            |
| tmp_vg.sh                                  | Shell script for run SRST2 to detect virulence factor genes      |
               





## Contacts

Ge Zhou  
PhD student  
**Email:** [ge.zhou@kaust.edu.sa](mailto:ge.zhou@kaust.edu.sa)  
**Lab:** Infectious Disease Epidemiology Lab  
**Institution:** King Abdullah University of Science and Technology (KAUST)

